src/challenges/challengeOne/5.Modal/Modal.test.tsx

```diff
@@ -1,8 +1,39 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Modal } from "./Modal";
-
-// describe("Modal", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Modal } from "./Modal";
+
+describe("Modal", () => {
+  it("renders modal with header and actions when isOpen is true", async () => {
```

Comment:
Again we need to separate these tests. Test the header, test the content, and props separately.